src/profession.cpp

```diff
@@ -289,11 +289,13 @@ void profession::check_definition() const
     if( !item_group::group_is_defined( _starting_items_female ) ) {
         debugmsg( "_starting_items_female group is undefined" );
     }
-    if( !_starting_vehicle.is_null() && !_starting_vehicle.is_valid() ) {
-        debugmsg( "vehicle prototype %s for profession %s does not exist", _starting_vehicle.c_str(),
-                  id.c_str() );
+    if( _starting_vehicle ) {
+        vproto_id vtype = *_starting_vehicle;
+        if( !vtype.is_valid() ) {
```

Comment:
`cata::optional` has a `operator->` so you can combine this into one line:
```suggestion
        if( !_starting_vehicle->is_valid() ) {
```
And include the check for the optional as well:
```suggestion
    if( _starting_vehicle && !_starting_vehicle->is_valid() ) {
```